vignettes/GOvis.Rmd

```diff
@@ -0,0 +1,497 @@
+library("multtest")
+library("Rgraphviz")
+library("hgu95av2.db")
+library("GO.db")
+library("genefilter")
+library("ALL")
+```
+
+The results reported in this document are based on
+`r packageDescription("hgu95av2.db")[["Version"]]` version of the
+`r Biocpkg("hgu95av2.db")` data package and on
+`r packageDescription("GO.db")[["Version"]]` of the `r Biocpkg("GO.db")` package.
+If you have different versions your results will probably be different.
+
+## Graphs
+
+The relationships between different GO terms, within a specific ontology
+are represented in the form of a directed acyclic graph. The leaves of
+this graph represent the most specific terms and their are edges from a
+specific term (child) to all less specific terms (each is a parent). The
+*induced* GO graph is the graph that obtains from taking a set of GO
+terms and finding all parents of those terms, and so on until the root
+node has been obtained. Given any set of genes one may use the mappings
+provided by GOA [@GOA] to find the most specific set of GO terms
+associated with those genes and hence, from those, to the induced GO
+graph. Thus we can speak of the induced GO graph either from the
+perspective of selected genes or from the perspective of selected terms.
+
+We note that there is a form of algebra possible on these induced GO
+graphs. If $G_1$ and $G_2$ are two different GO graphs from the same
+ontology then we can form their union, $G_1 \cup G_2$ which is the union
+of the nodes in the two graphs together with the union of the edge sets.
+We note that this is a bit different than the usual mathematical
+definition of the union of two graphs. Other set operations such as
+intersection and complement are also easy to define and implement and
+they will play a role in our subsequent discussions.
+
+## Some problems
+
+One should be aware that our state of biological knowledge is constantly
+changing and that the terms, their relationships and the genes mapped to
+them are constantly changing and being updated. The meta-data used to
+prepare this report will likely be out of date in less than one year.
+Hence, one must try to ensure that the mappings being used are up to
+date (Bioconductor meta-data packages are built quarterly).
+
+Another practical issue that needs some attention is consideration of
+the incompleteness of the data. Not all genes (or all transcripts) are
+assayed and not all genes are well enough understood to accurately
+annotate them at the existing GO nodes; to say nothing of the issues
+regarding the evolving nature of GO discussed above. For each problem
+the data analyst will need to consider what, if any, biases these issues
+might raise.
+
+In all comparisons and gene list developments reported in the remainder
+of this paper we restrict attention to those genes which satisfy two
+conditions first they were included on the chip being used and second
+they have a GO annotation for the ontology of interest.
+
+<a id="sec:example" ></a>
```

Comment:
I think this can be removed as well as any extra lines.

Reply:
we have one paragraph in visualization section in first paragraph where we use this label "sec:example" if we remove this line then how can we get back to this section after we click on "0.3". ? I have added this line for this purpose only please clarify.. should I remove this or leave this?

Reply:
Done.